### PR TITLE
Added `edge_lengths` property

### PR DESCRIPTION
**Pull request by janbridley**
## Description
Added edge_lengths property to the Polyhedron class.

## Motivation and Context
Although this is a single line of code, I use it extremely frequently (as it has additional value in helping to determine the degree of uniformity of a solid). This should be a quick PR, and if it seems unnecessary I am happy to continue manually implementing this function.

## Types of Changes

- [ ] Documentation update
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change<sup>1</sup>

<sup>1</sup>The change breaks (or has the potential to break) existing functionality.

## Checklist:

- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/coxeter/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/coxeter/blob/master/ContributorAgreement.md).
- [x] The changes introduced by this pull request are covered by existing or newly introduced tests.
- [ ] I have updated the [changelog](https://github.com/glotzerlab/coxeter/blob/master/ChangeLog.txt) and the [credits](https://github.com/glotzerlab/coxeter/blob/master/doc/source/credits.rst).

### CODE REVIEW

File: coxeter/shapes/polyhedron.py [hunk 1]
```diff
@@ -390,6 +390,11 @@ def edge_vectors(self):
         """:class:`numpy.ndarray`: Get the polyhedron's edges as vectors."""
         return self.vertices[self.edges[:, 1]] - self.vertices[self.edges[:, 0]]
 
+    @property
+    def edge_lengths(self):
+        """:class:`numpy.ndarray`: Get the length of each edge of the polyhedron."""
```
Comment (DomFijan) [inline coxeter/shapes/polyhedron.py:395]:
```suggestion
        """:class:`numpy.ndarray`: Get the length of each edge of the polyhedron in the same order as order of edges in :meth:`edge_vectors` ."""
```

**Review by DomFijan — CHANGES_REQUESTED** [review]
This looks good. Lets make sure that order of edge lengths is well defined in the docs and this is good to go.

File: coxeter/shapes/polyhedron.py (hunk 1 above)
Comment (janbridley) [reply coxeter/shapes/polyhedron.py:395]: Should we add this ordering specification to edge_vectors as well? Both use the same ordering as :meth:`edges` which is the primary sorted array

Comment (DomFijan) [reply coxeter/shapes/polyhedron.py:395]: Maybe. Is it super obvious what this is? If it is maybe this needs not be in the docstring, but if it is not then it should be for sure.

**Review by DomFijan — APPROVED** [review]

**Pull request merged**